topics/synthetic-biology/tutorials/basic_assembly_analysis/tutorial.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: tutorial_hands_on
+
+title: Designing plasmids encoding predicted pathways by using the BASIC assembly method
+zenodo_link: https://zenodo.org/record/6123887
+questions:
+- How to design plasmids encoding predicted pathways using BASIC assembly method ?
+- How to generate scripts enabling the automatic build of plasmids ?
+objectives:
+- Annotate the SBML pathway using Selenzy tool.
+- Design plasmids encoding predicted pathways using rpbasicdesign tool.
+- Generate dnabot scripts enabling the automatic build of plasmids.
+time_estimation: 15M
+key_points:
+- Annotation of SBML pathway using Selenzyme
+- Generating Constructs using rpbasicdesign
+- Generating python scripts using dnabot

```

Comment:
Could you rephrase these. They should be statements summing up the take-home messages. So ask yourself, if there are 3-5 things I hope the student will have learned from this, what is it?

So formulated not as "Doing task X", but "When doing X, always make sure to Y", or "Doing X is useful because Z" etc, real conclusions. The way these are phrased now, they are more objectives.

Reply:
Yes, understood, thank you for your explanation. We changed that.